prow/deck/jobs/jobs.go

```diff
@@ -155,56 +150,6 @@ func (ja *JobAgent) GetProwJob(job, id string) (kube.ProwJob, error) {
 	return j, nil
 }
 
-// GetJobLogTail returns the last n bytes of the job logs, works for both kubernetes and jenkins agent types.
-func (ja *JobAgent) GetJobLogTail(job, id string, n int64) ([]byte, error) {
```

Comment:
This and the underlying `GetLogTail` functions did not work -- the call to k8s API was to not limit the lines of text but to limit the bytes, so it would limit to the FIRST `n` bytes of the log.